packages/app/src/components/techdocs/TechDocsPage.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Content } from '@backstage/core-components';
+import {
+  TechDocsPageHeader,
+  TechDocsPage,
+  Reader,
+} from '@backstage/plugin-techdocs';
+import React from 'react';
+
+const DefaultTechDocsPage = () => {
+  return (
+    <TechDocsPage>
+      {({ techdocsMetadataValue, entityMetadataValue, entityId, onReady }) => (
+        <>
+          <TechDocsPageHeader
+            techDocsMetadata={techdocsMetadataValue}
+            entityMetadata={entityMetadataValue}
+            entityId={entityId}
```

Comment:
I'd say we want to call this `entity` or `entityName`, unless we serialized it to the ref format (`k:ns/n`), in which case it could be `entityRef`

Reply:
hmmm.. we are using `entityId` over a lot of different TechDocs components, thats why I kept that usage, do you think we should take the opportunity to replace it everywhere with something like `entityName`?

Reply:
Let's call it `entityRef`, even if it's the object form. Might be that we change it to the string form at some point but let's leave that to the future.

We're trying to get rid of "entity name" as a name for the kind+namespace+name triplet so `entityName` should be avoided. Entity ID is a bit too close to entity UID, which is a different thing. I also don't see much usage `entityId` in the public APIs of TechDocs, only the `Reader` prop, which we could migrate as well. Finally `entity` is best reserved for actual entities rather than refs.

Reply:
Alright, updating!

Reply:
I only updated the ones affected the components that has been changed in this PR (+ Reader prop), the rest we can do separately.